reference/halyard/secrets/s3-secrets.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: single
+title:  "Secrets in S3"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+
+This document describes how to set up Spinnaker secrets in an encrypted S3 bucket. In this example, we'll be using a `mybucket` bucket in the `us-west-2` region to store GitHub credentials and a kubeconfig file. We'll be referencing the bucket by its URL `mybucket.us-west-2.amazonaws.com`.
```

Comment:
```suggestion
This document describes how to set up Spinnaker secrets in an encrypted S3 bucket. This example uses a bucket (`mybucket`) in the `us-west-2` region to store GitHub credentials and a kubeconfig file. You reference the bucket by its URL `mybucket.us-west-2.amazonaws.com`.
```

Reply:
@KathrynLewis I know I'm killing a lot of "we"s. I think "we," "us," "let's," etc. are great in tutorials and such, but I think it's better to use 2nd person in most documentation.

Reply:
No problem! I agree it sounds better in 2nd person